src/app/config/config.c

```diff
@@ -7193,6 +7193,9 @@ parse_port_config(smartlist_t *out,
         } else if (!strcasecmp(elt, "KeepAliveIsolateSOCKSAuth")) {
           socks_iso_keep_alive = ! no;
           continue;
+        } else if (!strcasecmp(elt, "ExtendedErrors")) {
```

Comment:
We need to document this new option in the man page.

Reply:
Yup, will be there in the new branhc.